### PR TITLE
Updating landing page layout and content

### PR DESCRIPTION
**Pull request by romeboards**
This is the start of the refresh of the landing page, and entrypoint into incorporating more Bulma elements into the front-end. This first commit takes the app out of a big `.hero` and updates it to follow the Bulma convention:

```html
<body>
  <nav></nav>
  <section class="section">
    <div class="container">
      <h1 class="title">Section</h1>
      <h2 class="subtitle">
        A simple container to divide your page into <strong>sections</strong>, like the one you're currently reading
      </h2>
    </div>
  </section>
</body>
```

One challenge to address is that we're currently wrapped all of our app views in a `.container`, which prevents the possibility of a fluid full width view. Something to rethink going forward, also something to see what Bulma's opinion might be on the matter.

### CODE REVIEW

**Comment by toolness** [comment]
Nice!  So the main reason I stayed with the `hero` layout is actually because it seems to be the only layout that keeps a sticky footer.  Right now a footer actually only shows when we're in compatibility mode (which you can enable via the developer/hamburger menu on your local dev instance), but as you can see from this thumbnail, there's some empty space under the footer if the page doesn't have lots of content on it and the window is big:

> unsticky_footer

I noticed that someone in a Bulma Github issue (I forget which one) mentioned that `hero` was the only layout that fixed this problem, which is part of why I stayed with it, even though it is semantically odd since our layout isn't terribly heroic.

Other than that I noticed that the slide up/down animation that appears when clicking a hamburger/dropdown seems to be absent with this new layout, so that should be fixed before merging this too.

Anyhow this looks like a good start, I definitely like the fixed navbar in general.  I didn't enable it earlier partly because I was worried that really tiny phones might lose valuable screen real estate, but I also like how it visually anchors the page regardless of scroll position.

File: frontend/lib/pages/index-page.tsx [hunk 1]
```diff
@@ -16,6 +16,17 @@ export default class IndexPage extends React.Component<IndexPageProps> {
     return (
       <Page title="Technology for Housing Justice">
         <div className="content">
+
+          <section className="hero is-link is-fullheight is-fullheight-with-navbar">
+            <div className="hero-body">
+              <div className="container">
+                <h1 className="title">
+                  Fullheight hero with navbar
```
Comment (toolness) [inline frontend/lib/pages/index-page.tsx]: Er is this example copy or something?

**Comment by romeboards** [comment]
Yesss still a WIP at the moment

**Comment by toolness** [comment]
Ah cool, can you add `[WIP] ` to the beginning of the PR's name just to make that super explicit?  I should add a `CONTRIBUTING.md` and mention that...

**Comment by toolness** [comment]
Closing this in favor of #312 (which basically has the same code, it's just without the weird history).

**Pull request closed without merging**